redis/commands/core.py

```diff
@@ -1246,16 +1246,26 @@ def exists(self, *names):
 
     __contains__ = exists
 
-    def expire(self, name, time):
+    def expire(self, name, time, option=None):
```

Comment:
Recently we started adding support for type hints. It's a long slog, but what do you think of adding them here?